HealthcareSystem/GraphicalEditorServer/Controllers/EquipmentController.cs

```diff
@@ -100,6 +103,25 @@ public ActionResult DeleteEquipmentTransfer(int id)
             return Ok();
         }
 
+        /*
```

Comment:
Why is this code commented? If it will not be used in the future, l suggest deleting it, since the last sprint is done.

Reply:
I will uncomment this once Marko adds all required fields to the Equipment Relocation table and creates DTO.

Dialog for more details about Equipment Relocation doesn't have any fields because we have to add more fields to the database table so we can return the complete object with all necessary Info for display in the dialog.